README.md

```
@@ -32,6 +32,39 @@ sequenceDiagram
     Svelte (Frontend)-->>User: Displays output and feedback
 ```
 
+# 🛠 Contributing
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document
null

(MD025, single-title, single-h1)